src/main/java/frc/robot/Constants.java

```diff
@@ -50,6 +50,10 @@ public static final class IntakeConstants {
     public static final double PIVOT_MAX = 10;
     public static final double PIVOT_MIN = 0;
 
+    public static final double PIVOT_kP = 0.025;
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0;
```

Comment:
How was all of your values tested. You can use the Hardware Client for help on this

Reply:
I copy pasted those numbers from last year's charged up repo and just tested it in tele-op on Roomba. The arm was very precise and I saw no oscillations in the pivot. But, the encoder values in SmartDashboard started to drift from the arm's actual position as I continued to move the arm up and down in a single test. Kern's said we are using absolute encoders on this year's robot so we shouldn't see the same result.

Reply:
Just a note on the new robot these values may be different and should be different